packages/api/src/providers/ActivitySendStatusTelemetry/ActivitySendStatusTelemetryComposer.tsx

```diff
@@ -0,0 +1,35 @@
+import { useGetActivityByKey, useSendStatusByActivityKey, useTrackEvent } from '../../hooks';
+
+import usePrevious from '../../hooks/internal/usePrevious';
+

```

Comment:
Consider typing out the payload and exporting it. We can do it in another pull request.

Notes:
- We could/should grab the `SendStatus` somewhere

```ts
type TelemetrySendStatusChangePayload = {
  clientActivityID?: string;
  hasAttachment?: 'true' | 'false';
  key: string;
  prevStatus?: 'sending' | 'send failed' | 'sent';
  status: 'sending' | 'send failed' | 'sent';
  type?: string;
};
```

Reply:
add the type locally for now.